blog/README.md

```diff
@@ -0,0 +1,298 @@
+# CoreOS Layering
+
+In this blog we are going to explore the coreos-layering technology in order to
+inject kernel-modules to an OpenShift node.
+
+We will use the [driver-toolkit](https://github.com/openshift/driver-toolkit)
+as a base image in order to build the [simple-kmod](https://github.com/openshift-psap/simple-kmod)
+kernel module and then add the `.ko` to a new layer on top of the RHEL-CoreOS
+image that runs on the node.
+
+### Different methods of injecting kernel modules to a cluster
+
+* Day 0: Creating a custom RHCOS image for deployment including the kernel module
+* Day 2 MCO: Using the Machine Config Operator with a layered image (what we are showing in this blog)
+    - `MachineConfig` manifests can also be added to the cluster installation process. In this
+      case, it can also been seen as a day 1 operatrion but it is a more complex flow.
+* Day 2 KMM: Using the Kernel Module Management Operator
+
+Note: Future recommendations are subject to change as enhancements and features
+are added.
+
+### Backgroud - Customizable RHEL CoreOS
+
+Some OpenShift administrators asked for more flexibility for RHEL CoreOS
+customization in specific contexts. The RHCOS image layering feature is generally
+available in OpenShift 4.13.
+
+CoreOS-Layering is making containers *bootable* in OCP which allows customers to manage
+their OS images through containers.
+Using containers has become an industry standart and many tools have been developed
+arround it including pipelines, security checkers, smoke tests, deliverability and more.
+Therefore, allowing *bootable containers* bring a lot of benefits and ease of maintenance
+with it.
+
+We can now create a customized base image using a different kernel or any
+particular software and consume it in a given cluster.
+
+RHCOS image layering allows you to easily extend the functionality of your base
+image by layering additional images onto the base image.
+This layering does not modify the base image, and it creates a custom layered
+image that includes all base functionality and add-ons to specific nodes in the
+cluster.
+
+### Red Hat support for third party components
+
+![image](https://github.com/ybettan/coreos-layering-driver-toolkit/assets/29724509/b5d494bc-f6c6-41d0-96a7-638fb23e275c)
+
+For more details about the support model, please check the [support-model](https://access.redhat.com/third-party-software-support)
+for using third party components such as kernel modules.
+
+### Prerequisites
+
+* OCP 4.12 (tech preview) or later (4.13 is GA).
```

Comment:
I would add:  RHCOS layering is tech preview in OCP 4.12  and fully supported in OCP 4.13+.

Reply:
Done.